packages/augur-ui/src/modules/markets-list/components/markets-header.tsx

```diff
@@ -101,13 +115,35 @@ export default class MarketsHeader extends Component<
         <div>
           <h1>{headerTitle}</h1>
 
-          <FilterDropDowns
-            filter={filter}
-            sort={sort}
-            updateFilter={updateFilter}
-            history={history}
-            location={location}
-          />
+          <div>
+            <div className={Styles.MarketCardsFormat}>
+              View
+              <ViewSwitcher
+                handleClick={() => updateMarketsListCardFormat('classic')}
+                type={Classic}
+                selected={marketCardFormat === 'classic'}
+              />
+              <ViewSwitcher
+                handleClick={() => updateMarketsListCardFormat('expanded')}
```

Comment:
Lets use constants. I seem to be on a constant kick.

Reply:
Sure!